vendor/engines/secure_rooms/spec/features/manage_card_reader_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "Managing CardReaders" do
+  let(:facility) { create(:setup_facility) }
+  let(:secure_room) { create(:secure_room, facility: facility) }
+  let(:admin) { create(:user, :administrator) }
+  before { login_as admin }
```

Comment:
Can we do this as the lowest possible permission level? I think we settled on facility admins and directors, but I could be wrong.